core/domain/stats_services.py

```diff
@@ -29,43 +29,66 @@
 # Counts contributions from all versions.
 VERSION_ALL = 'all'
 
-def update_stats(exp_id, exp_version, state_name, event_type, update_params):
-    """Updates ExplorationStatsModel according to the incoming event model.
+def get_exploration_stats(exp_id, exp_version):
+    """Retrieves the ExplorationStatsModel and returns it in dict format.
```

Comment:
Let the client do to_frontend_dict(); this should return the domain object. Converting to a dict should only be done at the application boundary.

Reply:
Done.